helpers/withUserContainer.ts

```diff
@@ -0,0 +1,15 @@
+import { Context } from '../@types/helpers'
+
+export const withUserContainer =
```

Comment:
lets move this container into the container folder:
https://github.com/garageScript/c0d3-app/tree/master/containers

Reply:
Also, please make a followup PR to make another container to check if a user is an admin